modules/sensor/src/rgb-depth/realsense/vpRealSense2.cpp

```diff
@@ -393,15 +584,16 @@ void vpRealSense2::close() { m_pipe.stop(); }
 /*!
    Return the camera parameters corresponding to a specific stream. This
    function has to be called after open().
-   \param stream : stream for which camera intrinsic parameters are returned.
-   \param type : Indicate if the model should include distorsion parameters or not.
+   \param stream : Stream for which camera intrinsic parameters are returned.
+   \param type   : Indicates if the model should include distorsion parameters or not.
+   \param index  : Index of camera in T265 device, 1: Left  2. Right
```

Comment:
Maybe here you should complete `index` description with something like. For other cameras don't set this parameter or use -1.